docs/explanation/system-snaps/upower/index.md

```diff
@@ -0,0 +1,31 @@
+(explanation-system-snaps-upower-index)=
+# Upower
+
+UPower is an abstraction for enumerating power devices, listening to device events and querying history and statistics. Any application or service on the system can access the `org.freedesktop.UPower` service via the system message bus, as long as they connect using the appropriate interface to the upower snap.
```

Comment:
You should mention and link to the relevant snapd interface. Also I think it would be nice to have here the link to the dBus methods.